lib/trailblazer/activity/path.rb

```diff
@@ -1,3 +1,5 @@
+require "forwardable"
```

Comment:
This makes fails Travis in `trailblazer-macro` as well...I guess we need it then